packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -57,11 +57,11 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     this._warnConditionallyRequiredProps(
```

Comment:
@dzearing , would appreciate a close look here since it is such a core part of fabric.